### DISCUSSION TITLE
Can Uberproto be removed in Crow v4 to support Hermes?

### DISCUSSION THREAD

**Discussion by jamesholcomb** [feathersjs/feathers#2493] [General]
I would like to enable Hermes in react native apps using the v4 Crow client.  This was reported in  https://github.com/feathersjs/feathers/issues/2076.

Uberproto was removed in v5 Dove with this PR https://github.com/feathersjs/feathers/pull/2178

Is backporting the PR to v4 feasible @daffl?  Or do the existing dependencies make this unworkable?

I do plan on eventually migrating client and server to v5, but this would be a nice stopgap.

**Comment by daffl** [comment]
Unfortunately not since it is technically a breaking change (documented [here](https://dove.docs.feathersjs.com/guides/migrating.html#uberproto)).

**Comment by jamesholcomb** [comment]
I see...I suppose I could migrate client apps to v5 (staying at socket-io.client@2.3) for now.   I did some prelim testing and it seems to be compatible with the v4.5 server.

**Reply by daffl** [reply]
That is definitely an option. None of the communication APIs changed as long as you are using the same socket.io-client version as on the server.
